src/components/BookStatus/BookRecordsUsers/UserRecordsMobileModal/UserRecordsMobileModal.css

```diff
@@ -0,0 +1,14 @@
+.records-dialog.MuiDialog-root .MuiDialog-container {
+  align-items: flex-end;
+  background-color: rgba(247, 247, 249, 0.2);
```

Comment:
We should define colours in the hex format since we use that in the rest of the app.